integration-tests/helpers/index.js

```diff
@@ -242,9 +242,29 @@ function execHelper (command, options) {
  * @param {string[]} integrationTestsPaths
  * @param {string} [followUpCommand]
  */
+async function isolatedSandbox (dependencies, isGitRepo, integrationTestsPaths, followUpCommand) {
+  return createSandbox(dependencies, isGitRepo, integrationTestsPaths, followUpCommand, true)
+}
 
+/**
+ * @param {string[]} dependencies
+ * @param {boolean} isGitRepo
+ * @param {string[]} integrationTestsPaths
+ * @param {string} [followUpCommand]
+ */
+async function linkedSandbox (dependencies, isGitRepo, integrationTestsPaths, followUpCommand) {
+  return createSandbox(dependencies, isGitRepo, integrationTestsPaths, followUpCommand, false)
+}
+
+/**
+ * @param {string[]} dependencies
+ * @param {boolean} isGitRepo
+ * @param {string[]} integrationTestsPaths
+ * @param {string} [followUpCommand]
+ * @param {string} [isolated=true]
+ */
 async function createSandbox (dependencies = [], isGitRepo = false,
-  integrationTestsPaths = ['./integration-tests/*'], followUpCommand) {
+  integrationTestsPaths = ['./integration-tests/*'], followUpCommand, isolated = true) {
```

Comment:
We do not really need the default value, since it is explicitely passed through all the time.

I would probably change the arguments order to make the default values arguments at the end.